### PR TITLE
[React-Native-Update]Removed comments and unused imports

### PR DESCRIPTION
**Pull request by IvanSakharovskii**
added google maven to build repository. Checked that skinSDK library is building, tried to put it locally in sample apps, all works fine.

### CODE REVIEW

File: sdk/android/build.gradle [hunk 1]
```diff
@@ -13,6 +13,7 @@ buildscript {
 
 allprojects {
     repositories {
+        maven { url "https://maven.google.com" }
```
Comment (brol1dev) [inline sdk/android/build.gradle:16]: why do we need it? Isn't `google()` the same as this?

**Review by brol1dev — COMMENTED** [review]
Just have one question about the maven repo inclusion.

File: sdk/android/build.gradle (hunk 1 above)
Comment (IvanSakharovskii) [reply sdk/android/build.gradle:16]: From google documentation: 

```
 If you're using a version of Gradle lower than 4.1, you must instead use:
 maven {
     url 'https://maven.google.com'
 }
```

https://developer.android.com/studio/build/dependencies

**Review by brol1dev — APPROVED** [review]
LGTM

**Comment by akshaydch11** [comment]
LGTM

**Review by akshaydch11 — APPROVED** [review]

**Pull request merged**